cmd/maya-apiserver/spc-watcher/handler.go

```diff
@@ -214,7 +215,15 @@ func validate(spc *apis.StoragePoolClaim) (bool, error) {
 		return mutateSpc, errors.Errorf("aborting storagepool create operation for %s as no poolType is specified", spc.Name)
 	}
 
-	if !(poolType == string(apis.PoolTypeStripedCPV) || poolType == string(apis.PoolTypeMirroredCPV)) {
+	poolTypeMap := map[apis.CasPoolValString]bool{
```

Comment:
this is good improvisation over previous changes that were here.
As we done this, can I ask for one more?
Can you create this as global list/array in this package instead of local variable?

Reply:
@utkarshmani1997 Can we come up with a better naming. The current naming is very technical.